$OpenBSD$

Disable TLSv1.3 since the LibreSSL server is not ready.

XXX: When enabling TLSv1.3, restore the TLSv1.3 specific bits in redis.conf!

Index: src/tls.c
--- src/tls.c.orig
+++ src/tls.c
@@ -44,6 +44,7 @@
 #define REDIS_TLS_PROTO_TLSv1_3     (1<<3)
 
 /* Use safe defaults */
+#undef TLS1_3_VERSION
 #ifdef TLS1_3_VERSION
 #define REDIS_TLS_PROTO_DEFAULT     (REDIS_TLS_PROTO_TLSv1_2|REDIS_TLS_PROTO_TLSv1_3)
 #else
